lru/kv_test.go

```diff
@@ -0,0 +1,161 @@
+package lru
+
+import (
+	"fmt"
+	"testing"
+)
+
+// TestKV ensures the KV LRU Cache behaves as expected including limiting,
+// eviction of least-recently used entries, specific entry removal, and
+// existence tests.
+func TestKV(t *testing.T) {
```

Comment:
This doesn't seem to test the Lookup function at all, including the case I previously mentioned about ensuring values are  updated.